components/cart/CartOrder.module.scss

```diff
@@ -0,0 +1,10 @@
+@import "../../styles/variables.scss";
+
+.cartOrder {
+  border: 1px solid $dark-color;
```

Comment:
what does `$dark-color` mean? 
I recommend yo use names for colors like:
`primary, secondary, font, color, background`;

If you are going to change theme someday in future, `$dark-color` may have light color as value